test/Microsoft.AspNet.Mvc.FunctionalTests/TempDataTest.cs

```diff
@@ -186,10 +186,10 @@ public TempDataTest(MvcTestFixture<TempDataWebSite.Startup> fixture)
             // Act & Assert
             var exception = await Assert.ThrowsAsync<InvalidOperationException>(async () =>
             {
-                await Client.PostAsync("/Home/SetTempDataInvalidType", content);
+                await Client.PostAsync("/TempData/SetTempDataInvalidType", content);
             });
             Assert.Equal("The '" + typeof(SessionStateTempDataProvider).FullName + "' cannot serialize an object of type '" +
-                typeof(TempDataWebSite.Controllers.HomeController.NonSerializableType).FullName + "' to session state.", exception.Message);
+                typeof(BasicWebSite.Controllers.TempDataController.NonSerializableType).FullName + "' to session state.", exception.Message);
```

Comment:
We should delete this test. We don't need a functional test to verify that we throw an exception here.